client/components/purchase-detail/purchase-button.jsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+
+const PurchaseButton = ( {
```

Comment:
Note that we can't attach a ref to stateless functions. This shouldn't be a problem provided that we don't need a popover targeted to this component.